manifests/init.pp

```diff
@@ -1,5 +1,53 @@
 # == Class keepalived
 #
+# @param sysconf_dir
```

Comment:
Since `init.pp` is potentially the first contact with module by a reader, can you add informations about parameters ?